CLI API missing 'imports' module in cli.js?

I'm unable to execute `$ modular-css` on the command line without being met with a 'command does not exist' error. 

I've installed it both locally and globally. I tested the `browserify` command (both local/global also) in the same project as modular-css to make sure commands are actually getting through. `browserify` returns its options list, but `modular-css` gives me guff. 

Interesting enough, and this might be related, making a `node` call to their respective bin files:

`$ node node_modules/browserify/bin/cmd.js` Returns the browserify options list. 

`$ node node_modules/modular-css/bin/cli.js` Returns an error:
`Error: Cannot find module '../imports'`

Obviously I don't know if you've omitted `imports` intentionally, but from what I'm seeing, I assume postCSS is the `.process()` command? Anyways! I'm trying to get a simple `modular-css ./entry.css ./output.css` (or even just output to stdout) working. I've replicated it on both Mac and PC. Anything I might be missing here?

`package.json` doesn't have a `bin` field, so npm isn't generating the alias. I'll fix that.

The `Cannot find module '../imports' is bit-rot in the CLI, I'll fix that too.

--- a/bin/cli.js
+++ b/bin/cli.js
@@ -1,26 +1,36 @@
-/* eslint no-console:0 */
 "use strict";
 
-var fs = require("fs"),
+var fs   = require("fs"),
+    path = require("path"),
+    
+    mkdirp = require("mkdirp"),
+    map    = require("lodash.mapValues"),
+    
+    Processor = require("../").Processor,
 
-    imports = require("../imports"),
-
-    result, out;
-
+    processor;
+    
 // Update checking
 require("update-notifier")({ pkg : require("../package.json" ) }).notify({ defer : true });
 
-result = imports.process(process.argv[2]);
+processor = new Processor();
 
-if(!process.argv[3]) {
-    return console.log(result.css);
-}
+processor.file(process.argv[2]).then(function(out) {
+    /* eslint no-console:0 */
+    var file = process.argv[3],
+        css  = processor.css();
+    
+    if(!file) {
+        return console.log(css);
+    }
 
-out = process.argv[3];
+    mkdirp.sync(path.dirname(file));
 
-fs.writeFileSync(out, result.css, "utf8");
-fs.writeFileSync(
-    result.css + ".json",
-    JSON.stringify(result.exports),
-    "utf8"
-);
+    fs.writeFileSync(file, css);
+    fs.writeFileSync(
+        path.basename(file, path.extname(file)) + ".json",
+        JSON.stringify(map(processor.files, function(part) {
+            return part.compositions;
+        }), null, 4)
+    );
+});

--- a/bin/cli.js
+++ b/bin/cli.js
@@ -15,7 +15,7 @@ require("update-notifier")({ pkg : require("../package.json" ) }).notify({ defer
 
 processor = new Processor();
 
-processor.file(process.argv[2]).then(function(out) {
+processor.file(process.argv[2]).then(function() {
     /* eslint no-console:0 */
     var file = process.argv[3],
         css  = processor.css();

--- a/package.json
+++ b/package.json
@@ -7,6 +7,7 @@
   "bugs": {
     "url": "https://github.com/tivac/modular-css/issues"
   },
+  "bin": "./bin/cli.js",
   "scripts": {
     "test": "npm run lint && mocha",
     "mocha": "mocha",

